### PR TITLE
Enable Logging

### PR DESCRIPTION
**Pull request by jijoongmoon**
Modify codes and buld for Logging

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: api/capi/src/nntrainer.cpp [hunk 1]
```diff
@@ -62,7 +63,7 @@ static int nn_object(ml_nnmodel_h *model) {
   try {
     nnmodel->network = new Network::NeuralNetwork();
   } catch (const char *e) {
-    std::cerr << "Error: heap exception: " << e << std::endl;
+    ml_loge("Error:  heap exception: %s", e);
```
Comment (dongju-chae) [inline api/capi/src/nntrainer.cpp:66]: Q. intended additional space?

File: api/capi/src/nntrainer.cpp [hunk 2]
```diff
@@ -83,7 +84,7 @@ int ml_nnmodel_construct_with_conf(const char *model_conf, ml_nnmodel_h *model)
 
   std::ifstream conf_file(model_conf);
   if (!conf_file.good()) {
-    std::cerr << "Error: Cannot open model configuration file : " << model_conf << std::endl;
+    ml_loge("Error: Cannot open model configuration file : %s", model_conf);
```
Comment (dongju-chae) [inline api/capi/src/nntrainer.cpp:87]: Q. is `model_conf` always a valid pointer? (because conf_file.good() was failed)

Comment (jijoongmoon) [reply api/capi/src/nntrainer.cpp:87]: conf_file.good() will check the validation of model_conf.

File: api/capi/src/nntrainer.cpp (hunk 1 above)
Comment (jijoongmoon) [reply api/capi/src/nntrainer.cpp]: fixed.! Thanks.

File: api/capi/src/nntrainer.cpp (hunk 2 above)
Comment (dongju-chae) [reply api/capi/src/nntrainer.cpp:87]: I meant that if `model_conf` is nullptr, this log message may cause segmentation fault.
So, I wonder that the null check of model_conf was already done in a caller.

**Pull request merged**